io.github.go-musicfox.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>io.github.go-musicfox</id>
```

Comment:
the app-id is invalid.

https://docs.flathub.org/docs/for-app-authors/requirements#application-id

Reply:
I just went to https://pages.github.com/ and confirmed that GitHub users can create pages with the domain `https://username.github.io.`, so this should be a valid app id because `go-musicfox` is able to create a page at url `go-musicfox.github.io` through GitHub pages anytime.